voskcli/transcribe.py

```diff
@@ -206,6 +208,49 @@ def transcribe(inputFile, outputFile, model):
     # print(vtt.content)
 
 
+def match_language_to_model(lang):
+    '''Try mapping a specified language to a language model
+
+    :param lang: The language string to map
+    :type lang: str
+    :return: Path to language model
+    '''
+    # check if we have an old language dir for this language first
+    language_dir = f'/usr/share/vosk/language/{lang}/'
+    if os.path.isdir(language_dir):
+        print(f'Mapping language option to model {language_dir}')
+        return language_dir
+
+    # Map some common language codes
+    language_mappings = {
+            'eng': 'en',
+            'deu': 'de',
+            'ger': 'de',
+            'spa': 'es',
+            'zho': 'cn',
+            'fra': 'fr',
+            'rus': 'ru',
+            'por': 'pt',
+            'tur': 'tr'}
+    lang = language_mappings.get(lang, lang)
+
+    # Map to a 2 character code
+    lang = lang[:2]
+
+    # Try finding a matching module
+    modules = glob.glob(f'/usr/share/vosk/models/*-{lang}-*')
+    if not len(modules):
+        modules = glob.glob(f'./models/*-{lang}-*')
+
+    if not len(modules):
+        print('ERROR: Unable to map language to model')
+        sys.exit(1)
```

Comment:
You could consider raising an exception like in other parts of the code to avoid importing sys. Although I highly doubt that matters :)

Reply:
Fixed